global_helpers/panther_greynoise_helpers.py

```diff
@@ -0,0 +1,98 @@
+import ast
+import datetime
+
+from dateutil import parser
+from panther_base_helpers import deep_get
+
+
+class GreyNoiseBasic:
```

Comment:
Thinking about the class names here.  I think we need to either:
1) Have four classes that are `GreyNoiseNoiseBasic`, `GreyNoiseNoiseAdvanced`,`GreyNoiseRIOTBasic`, and `GreyNoiseRIOTAdvanced`. These would align with the four different lookup tables.
2) We keep two classes that are `GreyNoiseBasic` and `GreyNoiseAdvanced` and there is logic within the functions to work with the correct tables underneath.

Reply:
The four lookup tables should be named as follows, if I have them noted correctly:

```
greynoise_noise_basic
greynoise_riot_basic
greynoise_noise_advanced
greynoise_riot_advanced
```